File: server/models/Account.js
// models/Account.js

'use strict';

const mongoose = require('mongoose'),
        Schema = mongoose.Schema;

mongoose.Promise = global.Promise;

var accountSchema = mongoose.Schema({
  'pseudo'        : {type:String},
  'fb_id'         : {type:String, index:true},
  'google_id'     : {type:String, index:true},
  'reputation'    : {type:Number, default:0},
  'notif'         : {type:Boolean, default:false},
  'votes_spent'    : {type:Number, default:0},
  'votes'         : [{type: Schema.Types.ObjectId, ref: 'Vote'}]
}, {versionKey: false});

module.exports = mongoose.model('Account', accountSchema);

File: android/app/src/main/java/com/devmobile/ofait/models/Account.java
package com.devmobile.ofait.models;

import com.google.gson.reflect.TypeToken;

import java.lang.reflect.Type;
import java.util.List;

/**
 * Created by Jean-Noel on 17/10/2016.
 */
public class Account {

    public String _id;
    public String pseudo;
    public String google_id;
    public String fb_id;
    public int votesSpent;
    public int reputation;
    public List<Vote> votes;
    public int remaining_contents;
    public boolean notif;

    public static Type typeAnswerOf() {
        return new TypeToken<Answer<Account>>() {
        }.getType();
    }
}
